master/buildbot/db/pool.py

```diff
@@ -145,8 +146,8 @@ def _start(self):
 
     def _stop(self):
         self._stop_evt = None
+        threads.deferToThreadPool(self.reactor, self._pool, self.engine.dispose)
```

Comment:
does this really works?
I've been scratching my head for hours with this. looking at the code, I saw it was actually harmless, as this is in the stop flow.